README.md

```diff
@@ -243,6 +249,69 @@ Given a value `f`, one can access its type representative via the
 
     1. No parts of `a` should be checked
 
+### Alt
+
+A value that implements the Alt specification must also implement
+the [Functor](#functor) specification.
+
+1. `a.alt(b).alt(c)` is equivalent to `a.alt(b.alt(c))` (associativity)
+2. `a.alt(b).map(f)` is equivalent to `a.map(f).alt(b.map(f))` (distributivity)
+
+#### `alt` method
+
+```hs
+alt :: Alt f => f a ~> f a -> f a
+```
+
+A value which has a Alt must provide a `alt` method. The
+`alt` method takes one argument:
+
+    a.alt(b)
+
+1. `b` must be a value of the same Alt
+
+    1. If `b` is not the same Alt, behaviour of `alt` is
+       unspecified.
+    2. `a` and `b` can contain any value of same type.
+    3. No parts of `a`'s and `b`'s containing value should be checked.
+
+2. `alt` must return a value of the same Alt.
+
+### Plus
+
+A value that implements the Plus specification must also implement
+the [Alt](#alt) specification.
+
+1. `x.alt(A.pempty())` is equivalent to `x` (right identity)
+2. `A.pempty().alt(x)` is equivalent to `x` (left identity)
+2. `A.pempty().map(f)` is equivalent to `A.pempty()` (annihilation)
+
+#### `pempty` method
+
+```hs
+pempty :: Plus x => () -> x
+```
+
+A value which has a Plus must provide an `pempty` function on its
+[type representative](#type-representatives):
+
+    M.pempty()
+
+Given a value `x`, one can access its type representative via the
+`constructor` property:
+
+    x.constructor.pempty()
+
+1. `pempty` must return a value of the same Plus
+
+### Alternative
+
+A value that implements the Alternative specification must also implement
+the [Applicative](#applicative) and [Plus](#plus) specifications.
+
+1. `x.ap(f.alt(g))` is equivalent to `x.ap(f).alt(x.ap(g))` (distributivity)
+1. `M.pempty().ap(f)` is equivalent to `M.pempty()` (annihilation)
```

Comment:
In purescript it's in different order and i think it's correct, so I have opened issue [purescript-control#34](https://github.com/purescript/purescript-control/issues/34)